lib/model/channel.dart

```diff
@@ -78,6 +83,27 @@ mixin ChannelStore on UserStore {
     };
   }
 
+  /// Fetch topics in a channel from the server, only if they're not fetched yet.
+  ///
+  /// The results from the last successful fetch
+  /// can be retrieved with [getChannelTopics].
+  Future<void> fetchTopics(int channelId);
+
+  /// Pairs of the known topics and its latest message ID, in the given channel.
+  ///
+  /// Returns null if the data has never been fetched yet.
+  /// To fetch it from the server, use [fetchTopics].
+  ///
+  /// The result is guaranteed to be sorted by [GetStreamTopicsEntry.maxId]
+  /// descending, and the topics are guaranteed to be distinct.
```

Comment:
nit, omit needless words:

```suggestion
  /// The result is sorted by [GetStreamTopicsEntry.maxId] descending,
  /// and the topics are distinct.
```